ml_metadata/tools/mlmd_bench/stats.cc

```diff
@@ -65,22 +67,40 @@ void ThreadStats::Merge(const ThreadStats& other) {
   finish_ = std::max(finish_, other.finish());
 }
 
-void ThreadStats::Report(const std::string& specification) {
-  std::string extra;
+void ThreadStats::Report(const std::string& specification,
+                         WorkloadConfigResult* workload_summary) {
+  if (done_ == 0) {
+    LOG(ERROR) << "Current workload has not been executed even once!";
+    return;
+  }
+
+  double bytes_per_second = 0.0;
+  double microseconds_per_operation =
+      (accumulated_elapsed_time_ / absl::Microseconds(1)) / done_;
+
+  std::string rate;
+  // Not all workloads will transfer bytes in the process.
+  // Also, if `elapsed_seconds` equals to 0, `bytes_per_second` will remain as
+  // 0.
```

Comment:
in this case, bytes_per_second should be INFINITY? and we should not set the `workload_summary->set_bytes_per_second` , as it is not applicable

Reply:
Yes, my bad. The comment and the code is a little unmatched. What I mean is that, maybe it is not very reasonable to not set `bytes_per_second` of a `summary` simply because the elapsed time is less than 1 seconds(read operations are very fast in MLMD as I tested, and the elapsed time is very short under moderate number of operations.). Thus, I do a granularity refinement of the time first to the microsecond and back to second as below. wdyt?